placesautocomplete/src/main/res/values/pacv_attrs.xml

```diff
@@ -5,11 +5,13 @@
         <attr name="pacv_googleMapsApiKey" format="string"/>
         <attr name="pacv_adapterClass" format="string"/>
         <attr name="pacv_historyFile" format="string"/>
+        <attr name="pacv_languageCode" format="string"/>
```

Comment:
we are preserving language code in PlacesApi class.
isn't it better to do not preserve copy of that value on view too but retrieve it from PlacesApi?